src/components/pages/homepage/HomepageNewsPopup.tsx

```diff
@@ -0,0 +1,156 @@
+import { motion } from 'framer-motion'
+import { useEffect, useState } from 'react'
+import styled, { useTheme } from 'styled-components'
+
+import { deviceBreakPoints } from '../../../styles/global-style'
+import Badge from '../../Badge'
+import Button from '../../Button'
+import TextElement from '../../customPageComponents/TextElement'
+import PageSectionContainer from '../../PageSectionContainer'
+
+const HomepageNewsPopup = () => {
+  const theme = useTheme()
+  const [isMounted, setIsMounted] = useState(false)
+  const [isVisible, setIsVisible] = useState(true)
+
+  const handleClose = () => {
+    setIsVisible(false)
+  }
+
+  const handleOpen = () => {
+    setIsVisible(true)
+  }
```

Comment:
nit: could be simplified with
```suggestion
  const toggleVisibility = (visible: boolean) => setIsVisible(visible)`
```